setup.py

```diff
@@ -7,6 +7,10 @@
         "pytest", "configparser",
     ],
     # the following makes a plugin available to pytest
-    entry_points={"pytest11": ["name_of_plugin = integrationtest.integrationtest_nanorc"],
-                  },
+    entry_points={
+        "pytest11":[
+            "parallel_rc_plugin = integrationtest.integrationtest_nanotimingrc_nanorc",
```

Comment:
From my research, it seems that, when running an integration test, one of these `entry_points` will always need to then be disabled as they have conflicting code and thus we'd need to run for example `pytest -p no:parallel_rc my_integtest.py` to disable my plugin, when wanting to use the integrationtest_nanorc module. As far as I'm aware, there isn't a way to make the setting of the entry point conditional - is this true?
If it would be inconvenient for everyone wanting to run an integrationtest  to have to add the `-p no:parallel_rc` then one solution is to just comment it out?

Reply:
I've actually just had a thought, I could make an entirely separate integration timing tests  which includes all my changes but this would result in a lot of repeated code - particularly in the `python/integrationtest/dro_map_gen.py` file